plugins/imagespacingbox/plugin.js

```diff
@@ -0,0 +1,105 @@
+(function () {
+	'use strict';
+	CKEDITOR.plugins.add('imagespacingbox', {
+		requires: 'image',
```

Comment:
I think there's one thing we might be able to improve.

Here we `require` the `image` plugin because bellow [we're using](https://github.com/liferay/liferay-ckeditor/pull/126/files#diff-66f649f4e3ad8a5239ef948956905454R44) it's "language keys";  which means an additional request will be made, because since we [switched](https://github.com/liferay/liferay-ckeditor/pull/119) to the `image2` plugin, the `image` plugin is not bundled with `ckeditor.js` anymore.

You'd think that this additional request is made to `image.js` but since that plugin is not compatible with `image2.js`, because they have shared functionality (and mostly because of [this](https://github.com/ckeditor/ckeditor4/blob/eb2d11644a796cd13cd047cd55541430359f9317/plugins/image/plugin.js#L22-L26) line), what happens is that the request is made to `image2.js` which somehow doesn't make a lot of sense since `image2.js` is bundled in `ckeditor.js` 

It's more or less visible in the devtools, 

1. We `require` the `image` plugin but since it's not compatible with our bundled `image2` plugin we get this warning

![error01](https://user-images.githubusercontent.com/5572/91693670-acb82a00-eb6b-11ea-9d70-06d3345848d0.png)

2. So `image2` ends up being "required" 

![error02](https://user-images.githubusercontent.com/5572/91693834-e9842100-eb6b-11ea-8581-3c81f3bc4aff.png)

3. Which isn't really needed since `image2` is already bundled in our `ckeditor.js` build

![error03](https://user-images.githubusercontent.com/5572/91693923-0fa9c100-eb6c-11ea-9246-b1b3e48c9c45.png)

Ideally we probably want to copy existing `lang` directories/files that contain translated language keys to our "custom" plugins' directories or reuse language keys that are available

Reply:
hey @julien 
This was the only way I manage to use `image` plugin's translation keys, but I see it is not the right way to do it.
I will now work on copying `lang` files to `imagespacingbox` and push the code as soon as I can.

Reply:
Ok, so I copied language keys from `image` plugin to `plugins/imagespacingbox` to avoid using `requires:`image`` as before. 
I have built ckeditor, these keys are after build in `ckeditor-dev/plugins/imagespacingbox` directory and working as expected in the portal. 
<img width="1139" alt="lang" src="https://user-images.githubusercontent.com/25637907/91698548-33243a00-eb73-11ea-8ffe-d4ec09a03ad0.png">

Reply:
@mateomustapic ok that seems better :+1: 

However, everyone needs to be able to do this. So you probably need to edit `ck.sh` to make sure the `lang` directory from `ckeditor-dev/plugins/image` is copied to `plugins/imagespacingbox` before running the build 

Does that make sense?

Reply:
Yes, makes sense. 
Just wanted to confirm if this is what we want to achieve.